sematic/ui/packages/common/src/pages/RunDetails/RunSection.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from '@emotion/styled';
+import Box from '@mui/material/Box';
+import { useTheme } from '@mui/material/styles';
+import Typography, { typographyClasses } from '@mui/material/Typography';
+import MoreVertButton from 'src/component/MoreVertButton';
+import RunsDropdown from 'src/component/RunsDropdown';
+import theme from 'src/theme/new';
+import TagsList from 'src/component/TagsList';
+
+const Section = styled.section`
+    margin-bottom: ${theme.spacing(3)};
+`;
```

Comment:
This should be centrally defined for all sections and contain the top and bottom padding settings.

Reply:
Sure, for this purpose, introduced `<Section>` and `<Headline>` React components explicitly now.